<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QMainWindow" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>50</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_previous">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>50</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>50</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>50</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="just_playingTrack">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="playlistView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>581</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="count_tracks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>533</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Info">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mode_list">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>530</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reboot">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QComboBox" name="mode_list_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_update">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
